### PR TITLE
Symfony: Explain how to ignore some exceptions

### PR DESCRIPTION
**Pull request by alexislefebvre**
I took the code from the [package](https://github.com/getsentry/sentry-symfony/blob/master/UPGRADE-4.0.md) and adapted it.

### CODE REVIEW

File: src/platforms/php/guides/symfony/configuration/symfony-options.mdx [hunk 1]
```diff
@@ -104,3 +104,27 @@ class Sentry
     }
 }
 ```
+
+## Exclude some exceptions
```
Comment (imatwawana) [inline src/platforms/php/guides/symfony/configuration/symfony-options.mdx:108]:
```suggestion
## Exclude Some Exceptions
```

File: src/platforms/php/guides/symfony/configuration/symfony-options.mdx [hunk 2]
```diff
@@ -104,3 +104,27 @@ class Sentry
...
+
+This example show how to ignore not found exceptions, but you can ignore any exception you want.
```
Comment (imatwawana) [inline src/platforms/php/guides/symfony/configuration/symfony-options.mdx:110]:
```suggestion
This example shows how to ignore not found exceptions, but you can ignore any exception you want.
```

**Review by imatwawana — COMMENTED** [review]
Made some style and wording fixes, but this still needs the requested technical review.

**Comment by cleptric** [comment]
I'll look at this next week, thanks @imatwawana and @alexislefebvre

File: src/platforms/php/guides/symfony/configuration/symfony-options.mdx (hunk 2 above)
Comment (alexislefebvre) [reply src/platforms/php/guides/symfony/configuration/symfony-options.mdx]: Thanks @imatwawana !

**Pull request merged**